osgbuild/constants.py

```diff
@@ -112,6 +115,9 @@
     '3.6': ['el7', 'el8', 'el9'],
     'osg-3.6': ['el7', 'el8', 'el9'],
     '3.6-upcoming': ['el7', 'el8', 'el9'],
+    'osg-3.7': ['el8', 'el9'],
+    '3.7-main': ['el8', 'el9'],
+    '3.7-upcoming': ['el8', 'el9'],
     'internal': ['el7'],
```

Comment:
Unrelated but it looks like we had an `internal` for some reason? What's this for?

Reply:
It's for RPMs that we needed to be available in the build repos to satisfy build dependencies, but we didn't want to release to the public.  For example, at one point centos 8 had an issue where they had the cgroups libraries in their repos, but not the headers; we had to steal the headers RPMs from Fedora 29's Koji (!) so we could build Condor for EL8.

Reply:
That might be why we went with the name "devops" instead of "internal".

Reply:
How difficult would it be to change `devops` and `internal` to  `internal` and `build-internal`, respecitvely?

Reply:
Er. Then we'd have tags named osg-build-internal-build. How about extra-deps?

As for devops, I'd rather not mess up people's systems and workflows by renaming what we currently have, but starting with 3.7 we can go with internal (3.7-internal).